frontend/elements/src/core/menu/menu-tabs/menu-tab-title.ts

```diff
@@ -0,0 +1,61 @@
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import { nothing } from "lit-html";
+import "@elements/core/images/ui";
+
+export type TitleKind = ""
+    | 'background-image'
+    | 'color'
+    | 'overlay'
+    | 'text'
+    | 'image'
+    | 'audio';
+
+const STR_LABEL_LOOKUP: any = {
```

Comment:
would be slightly nicer to constrain the keys to TitleKind - but not a big deal :)